metadata.json

```diff
@@ -22,7 +22,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 6.1.0 < 8.0.0"
+      "version_requirement": ">= 6.1.0 < 9.0.0"
```

Comment:
I'd be hesitant to add this until it's actually available. I don't see it in on https://yum.puppet.com/ just yet.

Reply:
that's fine, I will revert this